### ISSUE TITLE
Add comments to projects

### ISSUE THREAD

**Issue by veraivanenko** [graphprotocol/mission-control-curator#314]
Hi. I think we could add comments feature. So, we could just comment the projects paying some fee. And the comment could be challenged.

**Comment by yanivtal** [comment]
Yes comments with something like https://medium.com/3box/introducing-3box-comments-plugin-c6a882d11f4a would be great.
